src/sass/layouts/hero.scss

```diff
@@ -0,0 +1,416 @@
+        
+        }
+        &-div {
+          display: block;
+          display: flex;
+      align-items: center;
+      flex-direction: column;
+      
+          
+        justify-content: center;
+        }
+        &-boxList{
+          width: 93px;
+          height: 34px;
+          border: 1px solid $second-text-color;
+          border-radius: 4px;
+          justify-content: space-between;
+          display: flex;
+          flex-direction: row;
+           align-items: center; 
+          padding: 10px;
+          box-sizing: border-box;
+          fill: $accent-color;
+          color:  $accent-color;
+
+          font-family: 'Poppins';
+font-style: normal;
+          font-weight: 700;
+font-size: 16px;
+line-height: 135%;
+          &:hover{
+
+            background-color: $accent-color;
+             fill: $light-primary-color;
+            color:  $light-primary-color;
+              }
+          }
+          &-select{
+            margin-top: 15px;
+            height: 84px;
+          overflow-y: scroll;
+          }
+          .is-hiden-select{
+            display: none;
+          }
+      .is-hden-select_display {
+        display: flex;
+      }
+          .hero-select::-webkit-scrollbar{
+            position: absolute;
+            transform: translateX(30px);
+            width: 2px;
+          
+          }
+    
+          .hero-list__ul {
+          // flex-direction: row;
+          border-radius: 4px;
+            border: 1px solid $accent-color;
+          }
+    
+          .hero-list__ulItem {
+       
+            width: 6.6px;
+          }
+     
+          .hero-select::-webkit-scrollbar-thumb{
+            background-clip: content-box;
+            height: 34px;
+            background: $second-text-color;
+            border-radius: 4px;
+          
+          }
+          &-item{
+            width: 83px;
+          height: 28px;
+          
+          font-family: 'Poppins';
+          font-style: normal;
+          font-weight: 700;
+          font-size: 10px;
+          line-height: 1.5%;
+          /* identical to box height */
+          
+          
+          display: flex;
+          flex-direction: row;
+          align-items: center;
+          padding: 4px 10px;
+          gap: 20px;
+       
+          color: $second-text-color;
+       
+          
+          
+          }
+          &-svg{
+    width: 20px;
+    height: 20px;
+ 
+    
+    &:hover {
+    fill: $light-primary-color;
+    }
+          }
+    
+      }
+
+      .coctails-section__coctails-img{
+    
+        width: 280px;
+        height: 308px;
+        background-size: cover;
+        background-image: url(../images/frame_desc@2x.png);
+       margin-bottom: 63px;
+       
+      }
+
+
+      }
+    
+    .is-hden{
+      display: none;
+    }
+  
+    .is-hover{
+      color: $accent-color;
+     }
+ 
+    .coctails-section-hover{
+      display: flex;
+      justify-content: center
+    }
```

Comment:
TODO:
1) use Prettier for whole file
2) rewrite media-requests according to project 'mixins'
3) delete commented lines
4) all images are retina optimized